deployment/terraform/cdn.tf

```diff
@@ -0,0 +1,75 @@
+resource "aws_cloudfront_distribution" "cdn" {
+  origin {
+    domain_name = "${module.origin.site_bucket}.s3.amazonaws.com"
+    origin_id   = "originEastId"
+  }
+
+  enabled             = true
```

Comment:
Minor, but when we create CloudFront distributions via the UI, IPv6 is usually enabled. We can enable it here via `is_ipv6_enabled`.